pkg/provider/azure.go

```diff
@@ -781,6 +783,7 @@ func (az *Cloud) getAzureClientConfig(servicePrincipalToken *adal.ServicePrincip
 		Authorizer:              autorest.NewBearerAuthorizer(servicePrincipalToken),
 		Backoff:                 &retry.Backoff{Steps: 1},
 		DisableAzureStackCloud:  az.Config.DisableAzureStackCloud,
+		UserAgent:               az.Config.UserAgent,
```

Comment:
Could you update the doc as well if we need to update the configuration in azure.json?

Reply:
this should be OK as the default value is still same as before

https://github.com/kubernetes-sigs/cloud-provider-azure/blob/0443bd214196ca00c06add9b9b31c5be63babf45/pkg/azureclients/armclient/azure_armclient.go#L88-L90